marlowe-chain-sync/app/Options.hs

```diff
@@ -0,0 +1,80 @@
+module Options
+  ( Options(..)
+  , getOptions
+  ) where
+
+import Cardano.Api (NetworkId (..), NetworkMagic (..))
+import Data.Maybe (fromMaybe)
+import qualified Options.Applicative as O
+import System.Environment (lookupEnv)
+import Text.Read (readMaybe)
+
+data Options = Options
```

Comment:
Eventually move to CLI component.

Reply:
_Maybe_ yes. See my other comment for more details.